File: examples/polyglot-app/cpp/patient.cpp
#include <iostream>
#include <string>

struct Patient {
  std::string rut;
  std::string email;
  std::string healthCondition;
};

void send(Patient user) {
  auto value = json["email"];
  std::cout << user.email;
  httpClient.post("https://clinic.example.com/patients", user.email);
}

File: examples/polyglot-app/rust/patient.rs
use serde::{Deserialize, Serialize};

#[derive(Serialize, Deserialize)]
struct Patient {
    rut: String,
    email: String,
    #[serde(rename = "birthDate")]
    birth_date: String,
    latitude: f64,
}

async fn send(user: Patient) {
    println!("{:?}", user.email);
    reqwest::Client::new()
        .post("https://clinic.example.com/patients")
        .send()
        .await;
}
